src/Appwrite/Platform/Workers/Functions.php

```diff
@@ -369,8 +378,13 @@ private function execute(
             $vars[$var->getAttribute('key')] = $var->getAttribute('value', '');
         }
 
+        $protocol = System::getEnv('_APP_OPTIONS_FORCE_HTTPS') == 'disabled' ? 'http' : 'https';
+        $hostname = System::getEnv('_APP_DOMAIN');
+        $endpoint = $protocol . '://' . $hostname . "/v1";
+
         // Appwrite vars
         $vars = \array_merge($vars, [
+            'APPWRITE_FUNCTION_ENDPOINT' => $endpoint,
```

Comment:
Name might be a bit misleading, maybe add API before endpoint.
```suggestion
            'APPWRITE_FUNCTION_API_ENDPOINT' => $endpoint,
```

or

```suggestion
            'APPWRITE_FUNCTION_API' => $endpoint,
```